packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -30,6 +30,65 @@ const WalletsCarouselContent: React.FC<TProps> = ({ onWalletSettled }) => {
     // for the embla "on select" callback
     // to avoid unbinding / cleaning etc, just let it use up-to-date list
     const walletsAccountsListRef = useRef(walletAccountsList);
+    const tweenNodes = useRef<HTMLElement[]>([]);
+    const tweenFactor = useRef(0);
+
+    const numberWithinRange = (number: number, min: number, max: number): number =>
```

Comment:
Define the type inside an object so it's easier to read and refer to 👍🏼

Reply:
Updated